### ISSUE TITLE
Bookmark Privately?

### ISSUE THREAD

**Issue by klantee** [rharel/webext-private-bookmarks#123]
Hello, the option to bookmark privately has been "greyed out" and relegated to the sub-menu within the Firefox search box???

**Comment by rharel** [comment]
Do you have the 'Disable outside of private browsing' option enabled?
If you have it enabled, is the button greyed out in a private window or a normal window?
If you have it enabled, did you give Private Bookmarks permission to run in private windows through Firefox as well? ![Capture](https://user-images.githubusercontent.com/3788700/63651824-817d4c80-c759-11e9-80c1-b7b8b8c47c77.PNG)

**Comment by klantee** [comment]
Yes to your first question. Button was greyed out in private and normal window. Private bookmarks had permission to run in private windows. (Update as from today) Since the bug occurred by disabling PB again, PB in private windows mode works again.  Will monitor for future reference.
